http/src/main/resources/reference.conf

```diff
@@ -504,14 +504,14 @@ clusterFiles {
 # so we need to ensure we don't use stale cache entries.
 runtimeDnsCache {
   cacheExpiryTime = 5 seconds
-  cacheMaxSize = 10000
+  cacheMaxSize = 1000
```

Comment:
Is there a reason to reduce these? I figure these caches don't use a ton of memory, and it's to be able to support a lot of users without evictions.

Reply:
<img width="770" alt="Screen Shot 2021-10-10 at 11 56 35 AM" src="https://user-images.githubusercontent.com/32771737/136703649-6e924243-c06e-4202-9938-c5f32da96d7a.png">

I looked at prod metrics over last 6 weeks, and for `runtimeDnsCache`, it seems maxed around 15 for size. So I think 1000 should be enough.

`googleTokenCache` has the largest size (see [dashboard](https://console.cloud.google.com/monitoring/dashboards/builder/813abbd0-604d-473c-a958-dd2dc205490d?authuser=0&project=broad-dsde-prod&dashboardBuilderState=%257B%2522editModeEnabled%2522:false%257D&timeDomain=6w) )...I'll make that larger then 1k